README.md

```diff
@@ -10,15 +10,31 @@ time-series data.  It is engineered up from PostgreSQL and packaged as a
 PostgreSQL extension, providing automatic partitioning across time and space
 (partitioning key), as well as full SQL support.
 
-If you prefer not to install or administer your instance of TimescaleDB, try the 
-30 day free trial of [Timescale](https://console.cloud.timescale.com/signup), our fully managed cloud offering. 
-Timescale is pay-as-you-go. We don't charge for storage you dont use, backups, snapshots, ingress or egress. 
+> [!WARNING]
+>
+>  The latest Postgres minor releases (17.1, 16.5, 15.9, 14.14, 13.17, 12.21), released today, have an unexpected
```

Comment:
I would use an explicit date instead of the word "today".

Reply:
Good point. updated.